### PR TITLE
DOCU-2803 - AppReg for multi RGs GA

### PR DESCRIPTION
**Pull request by acgoldsmith**
### Summary
changelog for DOCU-2803

### CODE REVIEW

File: app/konnect/updates.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## November 2022
 
+### 2022.11.21
+
+**Application Registration support in any runtime group**
+: {{site.konnect_short_name}} now officially supports the [app registration to services in both default and non-default runtime groups](/konnect/dev-portal/applications/enable-app-reg/#support-for-any-runtime-group). Portal developers will be able to register the application to consume services proxied through gateway services in both default and non default runtime groups.
+
+Currently, this feature is only available for {{site.base_gateway}} 3.0.0.0 or later.
```
Comment (acgoldsmith) [inline app/konnect/updates.md:19]:
```suggestion
:  Currently, this feature is only available for {{site.base_gateway}} 3.0.0.0 or later.
```

**Comment by acgoldsmith** [comment]
Will follow https://github.com/Kong/docs.konghq.com/pull/4806

File: app/konnect/updates.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## November 2022
 
+### 2022.11.21
+
+**Application Registration support in any runtime group**
+: {{site.konnect_short_name}} now officially supports the [app registration to services in both default and non-default runtime groups](/konnect/dev-portal/applications/enable-app-reg/#support-for-any-runtime-group). Portal developers will be able to register the application to consume services proxied through gateway services in both default and non default runtime groups.
```
Comment (lena-larionova) [inline app/konnect/updates.md:17]:
```suggestion
: {{site.konnect_short_name}} now officially supports [app registration to services in both default and non-default runtime groups](/konnect/dev-portal/applications/enable-app-reg/#support-for-any-runtime-group). Portal developers can register their applications to consume services proxied through gateway services in both default and non-default runtime groups.
```

File: app/konnect/updates.md (hunk 1 above)
Comment (lena-larionova) [reply app/konnect/updates.md:19]: (sorry, accidentally edited your edit, but that's my suggestion above)

"Currently" implies that this is going to change. Is that the case?

**Review by lena-larionova — CHANGES_REQUESTED** [review]

**Review by lena-larionova — APPROVED** [review]

File: app/konnect/updates.md [hunk 3]
```diff
@@ -11,6 +11,13 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## November 2022
 
+### 2022.11.21
+
+**Application registration support in any runtime group**
+: {{site.konnect_short_name}} now officially supports [app registration to services in both default and non-default runtime groups](/konnect/dev-portal/applications/enable-app-reg/#support-for-any-runtime-group). Portal developers can register their applications to consume services proxied through gateway services in both default and non-default runtime groups.
+
+:  Currently, this feature is only available for services being proxied through {{site.base_gateway}} 3.0.0.0 or later.
```
Comment (acgoldsmith) [inline app/konnect/updates.md:19]:
```suggestion
:  This feature is only available for services being proxied through {{site.base_gateway}} 3.0.0.0 or later.
```

**Pull request merged**